src/OmniSharp.LanguageServerProtocol/Logging/LanguageServerLogger.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using Microsoft.Extensions.Logging;
+using OmniSharp.Extensions.LanguageServer;
+using OmniSharp.Extensions.LanguageServer.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol;
+using OmniSharp.Logging;
+using OmniSharp.Stdio.Protocol;
+using OmniSharp.Stdio.Services;
+using OmniSharp.Utilities;
+
+namespace OmniSharp.LanguageServerProtocol.Logging
+{
+    class LanguageServerLogger : ILogger
```

Comment:
It looks like most of this is copied and pasted from another logger. Is it possible to unify these?

Reply:
probably!